Makefile

```diff
@@ -0,0 +1,33 @@
+SHELL=/bin/bash
+export PUBLISHER=seattle2030
+export PROJECT_NAME=ddns-updater
+ifeq ($(PUBLISHER),)
+    export TAG=$(PUBLISHER)/$(PROJECT_NAME)
+else
+	export TAG=$(PUBLISHER)/$(PROJECT_NAME)
+endif
+
+export COMMIT_ID:=$(shell git rev-parse --short HEAD)
+
+.PHONY:
+
+build:
+	docker build -t $(TAG):$(COMMIT_ID) .
+
+build-M1:
+	docker buildx build --push -t $(TAG):$(COMMIT_ID) -o type=image --platform=linux/arm64,linux/amd64 .
+
+example-run :
+	docker run --rm -e DOMAIN=seattle2030.net -e TYPE=A -e NAME=dev -e SECRET=Pe6qs93YTXiXbu2AVX4Qo5 -e KEY=eoWr51QvUCtx_WtywVr1meeDrD6Q52kjkg1 -e TTL=600 --name $(PROJECT_NAME) $(TAG):$(COMMIT_ID)
```

Comment:
Since the project is public, KEY and SECRET are visible to the outside. This is a test value, but will there be any issues?

Reply:
the KEY and SECRET is dummy. but It would be better to express the dummy more clearly in the variable name.

Reply:
I agree.

Reply:
I extracted the value to variable name with DUMMY.